proto/storage/service_identity.proto

```diff
@@ -19,19 +19,22 @@ message ServiceIdentity {
 
 // Next available tag: 13
 enum ServiceType {
```

Comment:
FYI: I've copied this changes to #8787 because they were required for TLS pre extension in the central reconciler.

Depending on timelines it might make sense to make those changes in an extra PR to get them in earlier.